src/linker/Linker/Annotations.cs

```diff
@@ -293,7 +293,22 @@ public TypePreserve GetPreserve (TypeDefinition type)
 
```

Comment:
`GetPreserve` can still return the accessibility bits - should it call `TryGetPreserve`? Same with `SetPreserve`, otherwise it could hit a case where the root assembly sets `TypePreserve.All | TypePreserve.AccessibilityVisible`, and an XML sets `TypePreserve.Methods` for example, and it would end up with just `TypePreserve.Methods`.

Reply:
I didn't notice as the method is never used. Remove it intead.